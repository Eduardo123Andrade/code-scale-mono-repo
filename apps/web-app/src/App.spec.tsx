import { it, expect } from "vitest"
import { render } from "@testing-library/react"
import App from "./App"

it("renders correctly", () => {
  const { asFragment } = render(<App />)

  expect(asFragment()).toMatchSnapshot()
})
