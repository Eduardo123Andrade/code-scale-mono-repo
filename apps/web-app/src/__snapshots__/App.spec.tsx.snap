// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Hello World
  </h1>
</DocumentFragment>
`;
